lib/Slider/Slider.js

```diff
@@ -898,6 +898,7 @@ module.exports = kind(
 	*/
 	previous: function () {
 		this.set('value', this.value - this._jumpIncrementAmount);
+		if (this.popup && this.$.popup.getShowing()) return true;
```

Comment:
Hi @aarontam 
Because we'd like to execute `spotSelect()`.
It shows popup when `onSpotlightSelect` event triggered.
If we return true every time, we cannot show popup using `onSpotlightSelect` event